cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/AivenApiService.java

```diff
@@ -38,16 +39,16 @@ public Map<String, String> createAcls(ClusterAclRequest clusterAclRequest) {
     String projectName = clusterAclRequest.getProjectName();
     String serviceName = clusterAclRequest.getServiceName();
 
-    LinkedHashMap<String, String> permissionsMap = new LinkedHashMap<>();
-    permissionsMap.put("topic", clusterAclRequest.getTopicName());
-    permissionsMap.put("permission", clusterAclRequest.getPermission());
-    permissionsMap.put("username", clusterAclRequest.getUsername());
+    Map<String, String> permissionsMap = new HashMap<>();
```

Comment:
here `EnumMap` should be better, it is much faster than `HashMap`

Reply:
@snuyanzin rest call needs "topic" as key for example. Either I can make Enums with "topic" as names which doesn't look so good.

Reply:
Maybe this might be a topic for another PR... have you checked https://fasterxml.github.io/jackson-databind/javadoc/2.5/com/fasterxml/jackson/databind/ser/std/EnumMapSerializer.html? I did not use it myself, however, it looks like Jackson already provides a way to serialize EnumMaps. If this assumption proves correct, you may actually get rid of HashMap in those situation, in which the EnumMap is preferred.

Reply:
Maybe this might be a topic for another PR... have you checked [Jackson's MapSerializer](https://github.com/FasterXML/jackson-databind/blob/92da0c813979936f792f99a779ae861d1ee3839e/src/main/java/com/fasterxml/jackson/databind/ser/std/MapSerializer.java)? I did not use it myself, however, it looks like Jackson already provides a way to serialize EnumMaps. If this assumption proves correct, you may actually get rid of HashMaps in those situation, in which an EnumMap is preferred.